troubleshooting.rst

```diff
@@ -0,0 +1,403 @@
+.. _troubleshooting:
+
+===============
+Troubleshooting
+===============
+
+This page lists approaches and mentions logs and metrics available
+to understand and debug Zeek's performance.
+
+There may be assumptions about Linux deployments regarding kernel features
+and tooling available.
+
+Memory Leaks and State Growth
+=============================
+
+When memory of any Zeek process continuously grows in production or testing
+settings, there might be a memory leak in Zeek's C++ core or it might be
+scripting state growth. Examples of the latter are a global table that is
+populated but elements aren't expired or removed. Containers attached to
+connections may also cause unbounded state growth when these connections
+are long lived.
+
+For such issues, using jemalloc's memory profiling can be invaluable. A good
+introduction to this topic is `Justin Azhoff's profiling presentation`_ (`slides`_).
```

Comment:
```suggestion
introduction to this topic is `Justin Azoff's profiling presentation`_ (`slides`_).
```

Reply:
Someday I'll get his name right. Looking at the github handle might have helped...